cnap/core/frame.py

```diff
@@ -58,6 +58,46 @@ def decrypt(self) -> None:
         """
         raise NotImplementedError("Subclasses should implement decrypt method.")
 
+class FrameZipBase(ABC):
```

Comment:
The description of this class should also be added to the docstring of this module.

Reply:
Fix all. Thanks!